atlassian-terraform/sumologic.auto.tfvars

```diff
@@ -2,11 +2,11 @@
 # Configure Sumo Logic credentials and App installation options.
 
 # Sumo Logic
-sumo_access_id          = "<YOUR SUMO ACCESS ID>"             # Please replace <YOUR SUMO ACCESS ID> (including brackets) with your Sumo Access ID. https://help.sumologic.com/Manage/Security/Access-Keys
-sumo_access_key         = "<YOUR SUMO ACCESS KEY>"            # Please replace <YOUR SUMO ACCESS KEY> (including brackets) with your Sumo Access KEY.
-deployment              = "us1"                               # Please update with your deployment, refer: https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
-sumo_api_endpoint       = "https://api.sumologic.com/api/v1/" # Please update with your sumologic api endpoint. Make sure the trailing "/" is present. Refer, https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
-app_installation_folder = "Atlassian"                         # The Sumo Logic apps will be installed in a folder specified by this value under your personal folder in Sumo Logic.
+sumo_access_id          = "<YOUR SUMO ACCESS ID>"         # Please replace <YOUR SUMO ACCESS ID> (including brackets) with your Sumo Access ID. https://help.sumologic.com/Manage/Security/Access-Keys
+sumo_access_key         = "<YOUR SUMO ACCESS KEY>"        # Please replace <YOUR SUMO ACCESS KEY> (including brackets) with your Sumo Access KEY.
+deployment              = "<YOUR SUMO DEPLOYMENT>"        # Please update with your deployment, refer: https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
+sumo_api_endpoint       = "<YOUR SUMOLOGIC API ENDPOINT>" # Please update with your sumologic api endpoint. Make sure the trailing "/" is present. Refer, https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
```

Comment:
Previous we have "/v1/" suffix in the example but in this referenced doc they don't have "v1". Do we need it here?

Reply:
Well, it was still expecting /v1 but the help page referred doesn't mention v1, so I have gone ahead and made changes to the script so the endpoint should be in the format 'https://api.sumologic.com/api/'. I also added an example so that it is more clear.